packaging/linux/openra.6

```diff
@@ -0,0 +1,68 @@
+.TH OPENRA 6
+.SH NAME
+openra \- An open-source implementation of the Command & Conquer: Red Alert engine using .NET/Mono and OpenGL.
+.SH SYNOPSIS
+.B openra
+[\fB\Game.Mod=\fR\fImodchooser\fR]
+.SH DESCRIPTION
+.B openra
+starts the game.
+.SH OPTIONS
```

Comment:
These will inevitably bitrot as people adjust the settings without realizing that they are manually documented here.

Could we please move these descriptions to `[Desc()]`s in the code, and then generate the man page using a simple utility command?